modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-relationships/request-organization-entry-to-request.json

```diff
@@ -0,0 +1,11 @@
+{
+	"deletionType": "prevent",
+	"label": {
+		"en_US": "Organization ID"
+	},
+	"name": "evpOrganizationId",
```

Comment:
Take out the "ID":
```suggestion
		"en_US": "Organization"
	},
	"name": "organization",
```

Reply:
Don't know why, but the curly brace and comma ("},") got red. Do not erase it, maintain it the way it is.